src/main/java/list_excercises/service/Compare.java

```diff
@@ -0,0 +1,42 @@
+package list_excercises.service;
+
+import java.util.ArrayList;
+
+public class Compare {
+
+    public double comparisson(ArrayList<String> arr1, ArrayList<String> arr2) {
+
+        int len1 = arr1.size();
+        int len2 = arr2.size();
+
+        if (len1 == 0 || len2 == 0) {
+            return 0.0;
+        }
+
+        double perc;
+        double trues = 0.0;
+        double falses = 0.0;
+
+        if (len1 >= len2) {
+            for (int i=0; i < len2; i++) {
+                if (arr1.get(i) == arr2.get(i)) {
```

Comment:
remember to use .equals() to compare the objects (== can be used only with the primitive types), eg.
(arr1.get(i).equals(arr2.get(i)))

Reply:
here you have some explanations: https://stackoverflow.com/questions/7520432/what-is-the-difference-between-and-equals-in-java